src/components/Header/Header.tsx

```diff
@@ -33,7 +33,13 @@ export const Header: React.FC<HeaderProps> = ({
   };
   return (
     <div className="Header" style={{ backgroundImage: `url(${PUBLIC_URL}/background.jpg)` }}>
-      <div role="button" className="SideBarOpenButton" onClick={sideBarToggle} aria-hidden="true">
+      <div
+        role="button"
+        id="sideBarOpenButton"
```

Comment:
do we need the id attribute here? if we do that in any component this makes impossible to use component twice on a page.